pom.xml

```diff
@@ -1,20 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>org.springframework.cloud</groupId>
-    <artifactId>spring-cloud-build</artifactId>
-    <version>1.1.0.RC1</version>
-    <relativePath />
-  </parent>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-build</artifactId>
+		<version>1.1.0.RC1</version>
+		<relativePath/>
+	</parent>
 
-  <groupId>org.springframework.cloud.stream.app</groupId>
-  <artifactId>spring-cloud-stream-app-starters</artifactId>
-  <version>1.0.0.BUILD-SNAPSHOT</version>
-  <packaging>pom</packaging>
+	<groupId>org.springframework.cloud.stream.app</groupId>
+	<artifactId>spring-cloud-stream-app-starters</artifactId>
+	<version>1.0.0.BUILD-SNAPSHOT</version>
+	<packaging>pom</packaging>
 
 	<dependencies>
```

Comment:
What is the purpose of these `<dependencies>` in the root POM?
Why `spring-cloud-stream-app-dependencies` isn't enough?
I just try to understand why do we need `SI-Java-DSL` in the `app-starters-common`. And yes, why only that dependency and not any other...
Thanks